src/openrct2/windows/cheats.c

```diff
@@ -609,13 +609,13 @@ static void window_cheats_guests_mouseup(rct_window *w, rct_widgetindex widgetIn
         window_cheats_set_page(w, widgetIndex - WIDX_TAB_1);
         break;
     case WIDX_GUEST_HAPPINESS_MAX:
-        game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_HAPPINESS, GAME_COMMAND_CHEAT, 255, 0);
+        game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_HAPPINESS, GAME_COMMAND_CHEAT, PEEP_MAX_HAPPINESS, 0);
         break;
     case WIDX_GUEST_HAPPINESS_MIN:
         game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_HAPPINESS, GAME_COMMAND_CHEAT, 0, 0);
         break;
     case WIDX_GUEST_ENERGY_MAX:
-        game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_ENERGY, GAME_COMMAND_CHEAT, 127, 0);
+        game_do_command(0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_ENERGY, GAME_COMMAND_CHEAT, PEEP_MAX_ENERGY, 0);
```

Comment:
Off-by-one? Effectively this changes from 127 to 128, while the 255 for happiness stays the same.